<!-- -->
<launch>

  <rosparam command="load" file="$(find lattice_planner)/config/params.yaml" />

  <node pkg="lattice_planner" type="lattice_planner" name="lattice_planner" output="screen">
    <remap from="/current_pose" to="/casper_auto/current_pose" />
    <remap from="/current_speed" to="/casper_auto/current_speed" />
    <remap from="/detected_objects" to="/casper_auto/detected_objects" />
    <remap from="/global_route" to="/casper_auto/global_route" />
    <remap from="/selected_path" to="/casper_auto/final_path" />
    <remap from="/final_waypoints" to="/casper_auto/final_waypoints" />
    <remap from="/generated_paths_marker" to="/casper_auto/generated_paths_marker" />
    <remap from="/obstacle_polygons" to="/casper_auto/obstacle_polygons" />
  </node>

</launch>
